docs/account/Change-password.mdx

```diff
@@ -0,0 +1,28 @@
+---
+id: change-password
+title: How to change password to your account
+hide_title: false
+sidebar_label: Change password
+slug: /account/change-password
+---
+
+import Tabs from "@theme/Tabs";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+### Change password through the App
+#### Loggen in
+
+- Go to My profile, then choose "Preferences & Settngs" and press "Change Password" button, enter your account's email. You will receive an email with a link to change your password.
+
```

Comment:
Please add here a yellow box here (e.g. like the one in the "Delete Account") that will mention that the email usually arrives in some minutes, a prompt to also check spam and if they receive no such email that they should contact support at:
[https://support.weatherxm.com/hc/en-us](https://support.weatherxm.com/hc/en-us)

Reply:
done!